src/pudl/transform/eiaaeo.py

```diff
@@ -171,12 +173,11 @@ def filter_enrich_sanitize(
         "Series and taxonomy must agree on case!"
     )
 
-    # TODO 2024-04-20: one day we'll include report year in the extract step -
-    # at that point we'll have to compare that with the report year extracted
-    # from the series name.
-    return sanitized.drop(columns=["region_series", "case_series"]).rename(
-        columns={"region_category": "region", "report_year_series": "report_year"}
-    )
+    # Dropping report_year_series here because we don't need it but we MIGHT want it later
+    # So not fully removing it from get_series_info()
```

Comment:
do we have a thing that checks whether report_year from extraction matches report_year_series?

Reply:
no, but I'm not too worried about it because there are only two years and it's pretty obvious if it's wrong.